bgpd/bgp_mplsvpn.c

```diff
@@ -1270,6 +1316,165 @@ void vpn_policy_routemap_event(const char *rmap_name)
 		vpn_policy_routemap_update(bgp, rmap_name);
 }
 
+void vrf_import_from_vrf(struct bgp *bgp, struct bgp *vrf_bgp,
+			 afi_t afi, safi_t safi)
```

Comment:
It would be helpful to know at a glance which struct bgp pointer is "from" and which is "to". Would you consider either adding "from" and "to" comments in the function parameter list, or using parameter names that include "from" and "to"?